### PR TITLE
Migrate to workflows

### PR DESCRIPTION
**Pull request by moggers87**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      - master
+  schedule:
+    - cron: '07 01 * * *'
+
+jobs:
+  unit-tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9, pypy3]
```
Comment (ad-m) [inline .github/workflows/tests.yml:18]: Are you aware that you can pass that value dynamically? 

See example: https://github.com/apache/airflow/blob/ecc3a4df0da67f258c3ad04733d6e561d8266c93/.github/workflows/ci.yml#L485

In this case, you can keep information about the supported Python version in one place, eg in tox.ini.

Comment (moggers87) [reply .github/workflows/tests.yml:18]: I'd have to write and maintain a script to extract that info from tox. No thanks.

**Comment by moggers87** [comment]
Still having issues with codecov. Even if it manages to upload a report, codecov tells me it has a missing reference.

**Comment by moggers87** [comment]
Screw it, code coverage is available in the CI output and Github Actions is not as horrid to use as Travis :laughing:

**Pull request merged**